pkg/watcher/watcher.go

```diff
@@ -0,0 +1,19 @@
+package watcher
```

Comment:
since the reload problem is now handled completely, maybe we can set the certificate [duration and renewBefore](https://github.com/k8tz/k8tz/blob/49764890ea04f5ed5b4a4edb1f9d040b26106567/charts/k8tz/values.yaml#L19-L20) values to cert-manager [default](https://cert-manager.io/docs/usage/certificate/#renewal) which is much shorter than our current default )that we set so long because of this problem that now solved)

Reply:
okay, I'll use the cert-manager default value for `duration` and `renewBefore`